docs/developer/README.md

```diff
@@ -61,9 +61,17 @@ Use an agent VM to test a device interaction, an image is automatically created
 hack/Containerfile.local and a qcow2 image is derived in output/qcow2/disk.qcow2, currently
 this only works on a Linux host.
 
+You can deploy a DB container of different sizes using a DB_VERSION variable for make command:
+* e2e (default) - minimal footprint for e2e testing
+* small - recommended setting for a demo environment 1000 devices max
+* demo-10k - recommended setting for a demo environment 10k devices max
```

Comment:
```suggestion
* demo-10k - recommended setting for a demo environment 10k devices max
```

Instead of demo-10k, I would call it medium-10k ?

Kind environment should not be used for demos, it's only ok for development, if we get customers understanding that they can demo on kind it has risks, I.e. the contents are volatile, when you bring down kind , all data is lost etc.

Can we change to:

small-1k
medium-10k

?
Then we always are explicit about the expected scale and not mention demos.